cli/endpoints/online/managed/vnet/setup/modules/vnet.bicep

```diff
@@ -0,0 +1,48 @@
+// Creates a virtual network
+@description('Azure region of the deployment')
+param location string = resourceGroup().location
+
+@description('Tags to add to the resources')
+param tags object = {}
+
+@description('Name of the virtual network resource')
+param virtualNetworkName string
+
+@description('Group ID of the network security group')
+param networkSecurityGroupId string
+
+@description('Virtual network address prefix')
+param vnetAddressPrefix string = '192.168.0.0/16'
+
+@description('private endpoint subnet address prefix')
+param scoringSubnetPrefix string = '192.168.0.0/24'
+
+resource virtualNetwork 'Microsoft.Network/virtualNetworks@2020-07-01' = {
+  name: virtualNetworkName
+  location: location
+  tags: tags
+  properties: {
+    addressSpace: {
+      addressPrefixes: [
+        vnetAddressPrefix
+      ]
+    }
+    subnets: [
+      { 
+        name: 'snet-scoring'
+        properties: {
+          addressPrefix: scoringSubnetPrefix
+          privateEndpointNetworkPolicies: 'Disabled'
```

Comment:
Only this needs to be disabled, below one "PrivateLinkServices" is not needed

Reply:
fixed